arwen/contexts/managedType.go

```diff
@@ -56,6 +58,19 @@ func NewManagedTypesContext(host arwen.VMHost) (*managedTypesContext, error) {
 	return context, nil
 }
 
+func initRandomizer(host arwen.VMHost) *math.SeedRandReader {
+	randomizer, _ := math.NewSeedRandReader(append(host.Blockchain().CurrentRandomSeed(), host.Runtime().GetCurrentTxHash()...))
```

Comment:
treat error here. or make NewSeedRandReader to not return error.

Reply:
it cannot return error here, that's why error is not treated. len(seed) > 0 in this specific use case.